src/models/ReturnSQSEvent.ts

```diff
@@ -5,7 +5,7 @@ export type EventType = "AUTH_IPV_AUTHORISATION_REQUESTED" | "F2F_YOTI_START" |
 export interface ReturnSQSEvent {
 	event_id: string;
 	client_id: string;
-	component_id: string;
+	clientLandingPageUrl: string;
```

Comment:
Even though we don't use it will we still get component_id (optionally) in our event payloads?

Reply:
we won't receive `component_id` as IPR not using that, the existing fields/events that IPR have asked for are configured here: https://github.com/alphagov/di-txma-event-self-serve/blob/main/config/consumer/sqs/return.json